books/computer-science/bake

```diff
@@ -48,7 +48,11 @@ recipe = Kitchen::BookRecipe.new(book_short_name: :computer_science) do |doc|
   book.chapters.each do |chapter|
     BakeLearningObjectives.v2(chapter: chapter, add_title: false)
     BakeNonIntroductionPages.v1(chapter: chapter)
-    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+
+    # create Chapter Review EOC wrapper
+    eoc_wrapper = ChapterReviewContainer.v1(chapter: chapter, metadata_source: metadata)
+
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper)
     BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata, klass: 'section-summary')
```

Comment:
should chapter summary also be appended to the wrapper?

Reply:
Good point-- the kitchen version doesn't currently have an append_to, so I'll make the change, and push that

Reply:
Maybe just use "Move custom section"?